idx/cassandra/cassandra_test.go

```diff
@@ -285,3 +387,46 @@ func BenchmarkLoad(b *testing.B) {
 	ix.Init()
 	ix.Stop()
 }
+
+func BenchmarkIndexingWithUpdates(b *testing.B) {
```

Comment:
our suggestion makes less sense.
the benchmark calls CasIdx.AddOrUpdate() with items that only update. So "BENCHMARK INDEXING WITH metricData payloads that result in UPDATES" == BenchmarkIndexingWithUpdates